### PR TITLE
Add orc file format

### PR DESCRIPTION
**Pull request by JkSelf**
Add orc file format.

### CODE REVIEW

File: proto/substrait/algebra.proto [hunk 1]
```diff
@@ -115,6 +115,7 @@ message ReadRel {
       enum FileFormat {
         FILE_FORMAT_UNSPECIFIED = 0;
         FILE_FORMAT_PARQUET = 1;
+        FILE_FORMAT_PARQUET = 2;
```
Comment (westonpace) [inline proto/substrait/algebra.proto:118]:
```suggestion
        FILE_FORMAT_ORC = 2;
```

**Review by westonpace — CHANGES_REQUESTED** [review]

**Comment by jacques-n** [comment]
I was thinking that we would change this structure per the comments on #169 and #183. @westonpace do you think you can push #169 forward? Then this could be layered in with a similar pattern.

**Comment by westonpace** [comment]
I should be able to find some time to work on this today.

File: proto/substrait/algebra.proto (hunk 1 above)
Comment (JkSelf) [reply proto/substrait/algebra.proto]: Thanks for your review. I have updated it to `FILE_FORMAT_ORC`.

**Comment by jacques-n** [comment]
Hi @JkSelf , this basic change was included using a new pattern in #169.

**Pull request closed without merging**